code/style.css

```diff
@@ -1,7 +1,490 @@
+  display: flex;
+
+  text-align: center;  
+  align-items: center;
+
+  position: relative;
+  
+  justify-content: center;
+  overflow: hidden;
+
+
+}
+
+.heading-text {
+  z-index: 2;
+  
+}
+
+video {
+  object-fit: cover;
+  background-size: contain;
+  background-repeat: no-repeat;
+  z-index: 0;
+  width: 100vw;
+}
+
+
+
+/* FAQ */
+
+article {
+  margin: 0.3rem;
+}
+
+#accordion {
+  flex: 1;
+  min-width: 100%;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(102, 12, 62);
+  padding: 5rem;
+
+}
+
+.questions-box {
+
+  display: flex;
+  flex-direction: column;
+  max-width: 70%;
+  align-self: center;
+
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
 .active + .answer {
-  /* some styling here */
+    /* some styling here */
+    display: block;
+    /* background-color: seashell; */
+
+}
+
+.question {
+
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 600;
+  font-size: 1.4rem;
+  color: #c23bce;
+}
+
+
+.answer {
+  align-self: center;
+  padding: 2rem 5rem 2rem 5rem;
+  color: #fd99d7;
+  display: none;
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+button {
+  background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
+}
+
+button:active {
+
+  color: red;
+}
+
+
+/* TEXT STYLING */
+
+h1 {
+  flex: 1;
+  font-family: 'Kumbh Sans', sans-serif;
+  margin: auto;
+  width: 100%;  
+  text-align: center;
+  font-size: 3.5rem;
+  font-weight: 600;
+  color: snow;
+  position: absolute;
+  margin-top: 13rem;
+
+}
+
+h2 {
+
+  margin: auto;
+  width: 100%;
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 300;
+  color: snow;
+  position: absolute;
+  padding: 5rem;
+  margin-top: 16rem;
+
+}
+
+h3 {
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 600;
+  font-size: 2rem;
+  color: rgb(252, 208, 252);
+  padding-bottom: 1.5rem;
+}
+
+
+
+
+/* FORM */
+
+#form-section {
+  font-family: 'Kumbh Sans', sans-serif;
+  font-size: 1.3rem;
+  font-weight: 300;
+  color: #fd99d7;
+  padding: 5rem;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-image: url("./images/guitar-rainbow.jpg");
+  background-size:cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.form-box {
+  padding: 3rem;
+  width: 80%;
+  background-color: rgba(27, 28, 77, 0.666);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4rem;
+  
 }
+
+.signup {
+
+  flex: 1;
+  min-width: 100%;
+}
+
+.right {
+
+  flex: 1;
+  max-width: 400px;
+  min-width: 310px;
+  
+}
+
+.left {
+  flex: 1;
+  max-width: 400px;
+
```

Comment:
maybe you could give right and left names that make you understand where they are in the html

Reply:
Yes, good idea.